moped-editor/src/views/projects/projectView/ProjectSummary/SubprojectLookupComponent.js

```diff
@@ -0,0 +1,54 @@
+import React, { useEffect, useRef } from "react";
+import { useGridApiContext } from "@mui/x-data-grid-pro";
+import { FormControl, FormHelperText, TextField } from "@mui/material";
+import Autocomplete from "@mui/material/Autocomplete";
+
+/** Component for subproject lookup dropdown
+ * @param {Integer} id - Data Grid row id (same as project id)
+ * @param {String} value - field value
+ * @param {String} field - name of field
+ * @param {Boolean} hasFocus - does this field have focus
+ * @param {Object} data - data object with subproject options
+ * @return {JSX.Element}
+ */
+
+const SubprojectLookupComponent = ({ id, value, field, hasFocus, data }) => {
```

Comment:
looks good—has there been any discussion of making a resuable autocomplete for datagrid? i see `FundAutocompleteComponent` that looks very similar, among others...

Reply:
Mateo and Rose mentioned it in dev sync yesterday, there are plans! https://github.com/cityofaustin/atd-data-tech/issues/20053